src/main/java/racing/RacingRule.java

```diff
@@ -0,0 +1,26 @@
+package racing;
+
+public class RacingRule {
+
+    private static int BASE_NUMBER = 4;
+
+    // 인스턴스화 방지
+    private RacingRule() {
+        throw new AssertionError();
+    }
+
+    public static int moveOrStop(int number) {
+        if(number >= BASE_NUMBER){
```

Comment:
```suggestion
        if (number >= BASE_NUMBER) {
```

코드 컨벤션을 지켜보는 것은 어떨까요?